test/e2e/replacement_values.py

```diff
@@ -24,5 +24,6 @@
     "TOPIC2": get_bootstrap_resources().Topic2.topic_arn,
     "KMSKEY": get_bootstrap_resources().KMSKey.key,
     "SNAPSHOT_CLUSTER_NAME1": get_bootstrap_resources().Cluster1.clusterName,
-    "SNAPSHOT_CLUSTER_NAME2": get_bootstrap_resources().Cluster2.clusterName
+    "SNAPSHOT_CLUSTER_NAME2": get_bootstrap_resources().Cluster2.clusterName,
+    "SNAPSHOT_CLUSTER_NAME3": get_bootstrap_resources().Cluster3.clusterName
```

Comment:
How long does it take to stand up 3 clusters? Is there any way to reduce this down to 1 - or at most 2? This seems really expensive and slow

Reply:
I don't think we can. Those clusters are made for validating snapshots. All tests run at the mean time. If they use same cluster, it has a problem that one test finishes and want to delete cluster for cleaning but another test is still using this cluster.

Reply:
I agree with @RedbackThomson, it is expensive to do here we can combine tagging validation as part of other snapshotting tests. For example one cluster can be snapshotted any number of times.

Reply:
@nmvk tagging validation test is for Tag and Untag APIs. I'm not sure whether we should merge it into other snapshot tests. Controller has tagging validation test for each resource. I don't think it is a good idea to only merge tags validation test for snapshot one into other snapshotting tests. If you really think it is better to reduce cluster numbers, then I will create a new PR for it.